packages/usuario/src/domains/usuarios.js

```diff
@@ -0,0 +1,35 @@
+const { Op } = require('sequelize');
+
+const findQuery = (usuarioModel) => async () => {
+  const rta = await usuarioModel.findAll({
```

Comment:
Could we give a better name to rta? I can't tell what is the purpose for this constant at first glance.